templates/build_script_go.sh

```diff
@@ -15,66 +15,115 @@
 #
 # ----------------------------------------------------------------------------
 
-PACKAGE_NAME={package_name}
-PACKAGE_VERSION={package_version}
-PACKAGE_URL={package_url}
+PACKAGE_NAME=${PACKAGE_NAME}
+PACKAGE_VERSION=${PACKAGE_VERSION}
+PACKAGE_URL=${PACKAGE_URL}
 
 yum -y update && yum install -y nodejs nodejs-devel nodejs-packaging npm python38 python38-devel ncurses git jq wget gcc-c++
 
-wget https://golang.org/dl/go1.16.1.linux-ppc64le.tar.gz && tar -C /bin -xf go1.16.1.linux-ppc64le.tar.gz && mkdir -p /home/tester/go/src /home/tester/go/bin /home/tester/go/pkg
+# Install Go and setup working directory
+wget https://golang.org/dl/go1.16.1.linux-ppc64le.tar.gz && \
+    tar -C /bin -xf go1.16.1.linux-ppc64le.tar.gz && \
+    mkdir -p /home/tester/go/src /home/tester/go/bin /home/tester/go/pkg
 
 export PATH=$PATH:/bin/go/bin
 export GOPATH=/home/tester/go
 
-OS_NAME=`python3 -c "os_file_data=open('/etc/os-release').readlines();os_info = [i.replace('PRETTY_NAME=','').strip() for i in os_file_data if i.startswith('PRETTY_NAME')];print(os_info[0])"`
+mkdir -p output
 
-mkdir -p /home/tester/$PACKAGE_NAME
+OS_NAME=`python3 -c "os_file_data=open('/etc/os-release').readlines();os_info = [i.replace('PRETTY_NAME=','').strip() for i in os_file_data if i.startswith('PRETTY_NAME')];print(os_info[0])"`
```

Comment:
Not a compulsory change, just a suggestion:
Can we replace this with :
`cat /etc/os-release | grep PRETTY_NAME | cut -d= -f2`
I think, we are doing same using  python.
As we are using shell script,  I guess above command might be more readable.